### DISCUSSION TITLE
不同分辨率显示器下的最佳体验

### DISCUSSION THREAD

**Discussion by wangkaifang** [ant-design/ant-design#34736] [Q&A]
我发现，antd pro 开发的项目，在显示器1920的分辨率下，可以得到最佳体验；
但是我们平时工作中，可能更多使用的是笔记本（如 macbook 13寸的显示器实际得到的是1440分辨率下的体验），因此，感觉各数据的展示略微的拥挤；
但是调整显示器的缩放 在 90% 的时候，刚好可以得到最佳体验；
于是，我在 html 标签上添加了全局的 样式， zoom: 90%，却发现了各组件的适配可能会存在略微的异常，而直接操作浏览器缩放的情况下没有；

因此，如果在1440以及更小的显示器分辨率下默认得到最佳体验呢？

**Comment by chloeeliu** [comment]
遇到了同样的问题，大部分用户用的是1440分辨率，ant的字体大小间距等，在1440体验不太好
